virttest/qemu_devices/qcontainer.py

```diff
@@ -1178,9 +1178,15 @@ def machine_arm64_mmio(cmd=False):
             """
             logging.warn('Support for aarch64 is highly experimental!')
             devices = []
+            if os.path.exists("/usr/share/edk2/aarch64/QEMU_EFI-pflash.raw"):
+                aavmf_code_file = "/usr/share/edk2/aarch64/QEMU_EFI-pflash.raw"
+                aavmf_vars_file = "/usr/share/edk2/aarch64/QEMU_EFI-pflash.raw"
+            else:
+                aavmf_code_file = "/usr/share/AAVMF/AAVMF_CODE.fd"
+                aavmf_vars_file = "/usr/share/AAVMF/AAVMF_VARS.fd"
             # EFI pflash
-            aavmf_code = ("-drive file=/usr/share/AAVMF/AAVMF_CODE.fd,"
-                          "if=pflash,format=raw,unit=0,readonly=on")
+            aavmf_code = ("-drive file=" + aavmf_code_file +
+                          ",if=pflash,format=raw,unit=0,readonly=on")
```

Comment:
Just a tip, "string" + variable + "string" is not a good way.
https://google.github.io/styleguide/pyguide.html#310-strings

Reply:
Thank you for your suggestion. I will change it later.